<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureWidget</class>
 <widget class="QWidget" name="MeasureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="current_calibration_factor">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>831</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: lightgreen; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Calibration Factor: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_valve_image">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Select Valve Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Binary</string>
   </property>
  </widget>
  <widget class="QLabel" name="original_valve">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>610</width>
     <height>330</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: brown;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="binarized_valve">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>220</y>
     <width>610</width>
     <height>330</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: brown;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="preview_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="measure_btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Measure</string>
   </property>
  </widget>
  <widget class="QLabel" name="valve_area_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>560</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 35px; color: black; color: brown; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Calculated Valve Area: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Threshold Value:</string>
   </property>
  </widget>
  <widget class="QSlider" name="threshold_value_slider">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="threshold_value_display">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;</string>
   </property>
   <property name="text">
    <string>127</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
